app/src/main/java/com/codesoom/assignment/enums/HttpMethodType.java

```diff
@@ -0,0 +1,15 @@
+package com.codesoom.assignment.enums;
+
+public enum HttpMethodType {
```

Comment:
이런 경우에는 이렇게 RFC 문서도 함께 읽을 수 있도록 링크해 줍시다.
```suggestion
/**
 * @see https://www.rfc-editor.org/rfc/rfc7231
 */
public enum HttpMethodType {
```

Reply:
앞으로 관련 문서를 주석으로 달아보겠습니다! 감사합니다! (a93872a5)

Reply:
RFC 7231 문서는 HTTP 스펙이므로 백엔드 개발자라면 개요라도 파악해둬야 하는 중요한 문서 중 하나입니다. 시간이 나실 때마다 꼭 군데군데 읽어보시고, 제대로 읽고 싶다면 "HTTP 완벽 가이드" 책을 구해서 공부해 보세요.

Reply:
오 안그래도 최근에 HTTP 완벽 가이드를 구입했었는데 빨리 읽어봐야겠습니다
감사합니다 :)